src/main/resources/templates/db_scripts/schema.json

```diff
@@ -90,6 +90,11 @@
       "run": "after",
       "snippetPath": "migration/update_budget_rollover.sql",
       "fromModuleVersion": "mod-finance-storage-8.4.0"
```

Comment:
to apply change in budget_encumbrances_rollover sql we need to change here to mod-finance-storage-8.7.0

Reply:
I updated `budget_encumbrances_rollover.sql` there is already mentioned version 8.7.0. No touche to `update_budget_rollover.sql`

Reply:
I updated `budget_encumbrances_rollover.sql` there is already mentioned version 8.7.0. No touch to `update_budget_rollover.sql`

Reply:
got it, it makes sense